### PR TITLE
[docs] Update Repo Documentation for Google Cloud Pub Sub Target

### PR DESCRIPTION
**Pull request by JeffNeff**

### CODE REVIEW

File: docs/targets/googlecloudpubsub.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Event Target for Google Cloud Pub Sub
```
Comment (tzununbekov) [inline docs/targets/googlecloudpubsub.md:1]: Here and below: the product name is "Google Cloud **Pub/Sub**", so let's stick to it.

```suggestion
# Event Target for Google Cloud Pub/Sub
```

File: docs/targets/googlecloudpubsub.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to a pre-defined Google Cloud Pub Sub Topic.
+
+
+## Prerequisite(s)
+
+- Google Cloud Console account.
+- A service account and it's associated JSON credentials.
+- a pre-existing Google Cloud Pub Sub Topic.
```
Comment (tzununbekov) [inline docs/targets/googlecloudpubsub.md:10]: Consistent styling:
```suggestion
- A pre-existing Google Cloud Pub/Sub Topic.
```

File: docs/targets/googlecloudpubsub.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Deploying an Instance of the Target
+
+1. [Clone][clone] the Triggermesh mono repo and navigate to the example manifest located in the [`triggermesh/config/samples/targets/googlecloudpubsub/`][sample-manifest] section of the Triggermesh mono repo.
```
Comment (tzununbekov) [inline docs/targets/googlecloudpubsub.md:15]: Here and below - camel casing:
```suggestion
1. [Clone][clone] the TriggerMesh mono repo and navigate to the example manifest located in the [`triggermesh/config/samples/targets/googlecloudpubsub/`][sample-manifest] section of the TriggerMesh mono repo.
```

Comment (tzununbekov) [reply docs/targets/googlecloudpubsub.md:15]: Can we also replace `mono repo` with something that does not expose unnecessary technical details of the project? We never used the term `mono repo` for user-facing docs.

File: docs/targets/googlecloudpubsub.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+```
+kubectl apply -f triggermesh/config/samples/targets/googlecloudpubsub/
+```
+
+## Event Types
+### Arbitrary
+This target consumes events of any type.
+
+[ce]: https://cloudevents.io/
+[ce-jsonformat]: https://github.com/cloudevents/spec/blob/v1.0/json-format.md
```
Comment (tzununbekov) [inline docs/targets/googlecloudpubsub.md:31]: Not used.

**Review by tzununbekov — CHANGES_REQUESTED** [review]

**Comment by jmcx** [comment]
can we archive this? I think we already pushed the doc to the doc website

**Pull request closed without merging**